### PR TITLE
Chore: only use source maps when running unit tests in debug mode

### PR DESCRIPTION
**Pull request by jeremypress**
Source maps don't work in phantom, so we should turn them off unless running our tests in debug mode in a browser. This should shave off between 30-45 seconds every place we run our tests; locally, jenkins, and travis.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jeremypress has signed the CLA. Thanks for the pull request!

File: build/webpack.karma.config.js [hunk 1]
```diff
@@ -1,19 +1,28 @@
 require('babel-polyfill');
 
+const { argv } = process;
```
Comment (jeremypress) [inline build/webpack.karma.config.js]: Looking for a better way to determine the original command to run  the tests

File: build/webpack.karma.config.js [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
-    devtool: 'inline-source-map',
     resolve: {
         alias: {
             sinon: 'sinon/pkg/sinon'
         }
     }
 });
 
+if (isDebug) {
+    config.devtool = 'inline-source-map';
```
Comment (jeremypress) [inline build/webpack.karma.config.js]: Will change this to the other source map type once I check it on all of our browsers

File: build/webpack.karma.config.js (hunk 1 above)
Comment (pramodsum) [reply build/webpack.karma.config.js]: We do this in webpack.config.js, would it work here? 
```
const isRelease = process.env.NODE_ENV === 'production';
const isDev = process.env.NODE_ENV === 'dev';
```

Comment (jeremypress) [reply build/webpack.karma.config.js]: That's a good catch to turn them off for prod, but we also want to turn them off if we're not running 'debug' in any environment. We never run debug during a production build so it should already be covered.

File: build/webpack.karma.config.js (hunk 2 above)
Comment (jeremypress) [reply build/webpack.karma.config.js]: changed

File: build/webpack.karma.config.js (hunk 1 above)
Comment (pramodsum) [reply build/webpack.karma.config.js]: ah good point.

**Pull request merged**